packages/demo/backend/src/config/assets.ts

```diff
@@ -0,0 +1,41 @@
+import type { Asset } from '@eth-optimism/verbs-sdk'
+import { base, baseSepolia, mainnet, unichain } from 'viem/chains'
+
+export const USDC: Asset = {
+  address: {
+    [mainnet.id]: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
+    [unichain.id]: '0x078d782b760474a361dda0af3839290b0ef57ad6',
+    [baseSepolia.id]: '0x036CbD53842c5426634e7929541eC2318f3dCF7e',
+  },
+  metadata: {
+    decimals: 6,
+    name: 'USDC',
+    symbol: 'USDC',
+  },
+  type: 'erc20',
+}
```

Comment:
question - should the verbs sdk just export a `USDC` asset and the user can use that? It feels like a lot of friction and potential room for error to have a dev need to define a commonly used asset like USDC

Reply:
yes, we should have the sdk export all major tokens. I added this [ticket](https://github.com/ethereum-optimism/verbs/issues/122) for that work.